ui/apps/platform/src/Containers/Network/Header/ClusterSelect.tsx

```diff
@@ -44,19 +43,16 @@ const ClusterSelect = ({
             placeholderText="Select a cluster"
             onSelect={changeCluster}
         >
-            {clusters
-                .filter((cluster) => networkGraphClusters[cluster.type])
```

Comment:
why is this filter no longer needed?

Reply:
The base (and bad) reason is that the piece of data needed by the filter is not available in the new API.
Looking closer at the code, the filter only excludes the `GENERIC_CLUSTER` type (and possibly SWARM clusters, not supported anymore since Jan 19 or DOCKER_EE clusters not supported anymore since Jun 19).
Manual cluster addition in the UI only allows choice between Kubernetes, OpenShift 3.x compatibility or OpenShift 4.x. All these cluster flavours pass the filter.

Reply:
Based on conversation [here](https://redhat-internal.slack.com/archives/CELUQKESC/p1677590535559309) let's create a follow up JIRA to error on `GENERIC_CLUSTER` when met with this value

Reply:
https://issues.redhat.com/browse/ROX-15783